src/main/java/org/venice/beachfront/bfapi/services/IABrokerService.java

```diff
@@ -12,6 +13,8 @@
 import org.springframework.web.util.UriComponentsBuilder;
 import org.venice.beachfront.bfapi.model.Scene;
 
+import com.fasterxml.jackson.databind.JsonNode;
+
 @Service
 public class IABrokerService {
 	private static final int ASYNC_POLL_INTERVAL_SEC = 10;
```

Comment:
Suggest making these two poll values into configuration values in application.properties